src/OmniSharp.Host/Services/OmniSharpEnvironment.cs

```diff
@@ -38,6 +40,20 @@ public class OmniSharpEnvironment : IOmniSharpEnvironment
             TraceType = traceType;
             TransportType = transportType;
             OtherArgs = otherArgs;
+
+            // On Windows: %APPDATA%\OmniSharp\omnisharp.json
```

Comment:
Should this be "%USERPROFILE%\.omnisharp\omnisharp.json" to match more closely to the Unix location? Usually, users aren't expected to edit or place files under %APPDATA%.